ckanext/example_ivalidators/tests/test_ivalidators.py

```diff
@@ -33,8 +28,28 @@ def test_overridden_validator(self):
         assert u"Hola cómo estás" == v("Hola c\xf3mo est\xe1s")
 
 
+@pytest.mark.usefixtures("with_plugins")
 class TestNoIValidators(object):
     def test_no_overridden_validator(self):
         v = get_validator("unicode_only")
         with pytest.raises(Invalid):
             v(b"Hola c\xf3mo est\xe1s")
+
+
+@pytest.mark.ckan_config("example.ivalidators.number", "10")
+@pytest.mark.usefixtures("with_plugins")
+def test_validator_used_by_declaration(ckan_config):
+    assert ckan_config["example.ivalidators.number"] == "10"
+
+    with pytest.raises(UnknownValidator):
+        # call get_validator with any value to build validators cache. Without
+        # this line, the test has no sense, because custom validators are
+        # blocked exactly by existing cache.
+        get_validator("negate")
+
+    plugins.load("example_ivalidators")
+
+    try:
+        assert ckan_config["example.ivalidators.number"] == -10
```

Comment:
Because this option declared in `ckanext/example_ivalidators/config_declaration.yaml` as `int_validator negate`: first validator parses `10`, second validator changes its sign